src/app/public/modules/file-attachment/file-drop.component.ts

```diff
@@ -32,6 +32,9 @@ export class SkyFileDropComponent {
   @Output()
   public filesChanged = new EventEmitter<SkyFileDropChange>();
 
+  @Output()
+  public linkInputBlurred = new EventEmitter<void>();
+
```

Comment:
Should we complete this emitter on destroy?

Reply:
Probably. Done. I also added completions for the other two events that didn't previously have that.

Reply:
Thanks!